CMakeLists.txt

```diff
@@ -20,26 +20,20 @@ else()
   message(ERROR " Unknown value for CMAKE_BUILD_TYPE: ${CMAKE_BUILD_TYPE}")
 endif()
 
-set(CMAKE_MODULE_PATH
-${CMAKE_CURRENT_SOURCE_DIR}/cmake
-)
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/Modules)
 
 find_package(PythonInterp 3.2 REQUIRED)
 message(STATUS "PYTHON_EXECUTABLE = ${PYTHON_EXECUTABLE}")
 
 #setup
-include(FindCXX11)
-include(FindOpenCL)
+find_package(CXX11 REQUIRED)
```

Comment:
I doubt you'll need this any more, honestly. For all our supported platforms, it suffices just to set the appropriate flag.